packages/fxa-settings/src/lib/gql.ts

```diff
@@ -36,20 +79,10 @@ export const errorHandler: ErrorHandler = ({ graphQLErrors, networkError }) => {
       }
     }
   }
-  if (networkError && 'statusCode' in networkError) {
-    if (networkError.statusCode === 401) {
-      reauth = true;
-    }
-  }
-  if (reauth && currentPageRequiresAuthentication) {
-    window.location.replace(
-      `/signin?redirect_to=${encodeURIComponent(window.location.pathname)}`
-    );
-  } else {
-    if (!reauth) {
-      console.error('graphql errors', graphQLErrors, networkError);
-    }
+  if (networkError) {
+    sentryMetrics.captureException(networkError);
   }
+  console.error('graphql errors', graphQLErrors, networkError);
```

Comment:
Do we want this here? If so, maybe a lot per error type. If not, remove it.

Reply:
At first, I removed this because I'm pretty sure we can see the same thing from the GQL endpoint response. However, I've also seen this be handy in the console a couple of times, so I left it. I'll do a log per type for now👍

Reply:
Personally I like it. It also will show up in client side breadcrumbs if an error is thrown. Maybe just two logs then? One for gql one for errors?